src/Lean/Server/InfoUtils.lean

```diff
@@ -316,7 +316,7 @@ where
         if let some ldecl := (← getLCtx).findFVar? e then
           !ldecl.userName.hasMacroScopes
         else false
-      else isAtomicFormat eFmt
+      else !e.isMVar
```

Comment:
Are you sure that you want to check for `isMVar` here? What about applications of an MVar to another MVar, for example?

Reply:
My thinking is while showing something like `?m : SomeType` doesn't really provide any useful information, it's better to err on the side of showing more information about inference for more complicated types.

Applying an MVar to an MVar would come from something like `_ _`, right? I think it would be useful to be able to explicitly see when Lean isn't able to infer terms for MVars in that kind of expression.